app/models/enterprise.rb

```diff
@@ -132,7 +132,7 @@ class Enterprise < ActiveRecord::Base
       where(nil)
     end
   }
-  scope :is_primary_producer, -> { where(is_primary_producer: true) }
+  scope :is_primary_producer, -> { where("enterprises.is_primary_producer IS TRUE") }
```

Comment:
I believe `where(enterprises: { is_primary_producer: true })` would work as well FYI. It doesn't make much difference here so I don't mean to change it.

Reply:
I don't remember the exact details, but i think there were some really nasty Postgres errors around this. I possibly explain it in the commit message...